core-bundle/src/Resources/public/sectionwizard.js

```diff
@@ -0,0 +1,101 @@
+(function(){
+    'use strict';
+
+    const initialized = [];
+
+    const init = (node) => {
+        node.querySelectorAll('.tl_sectionwizard').forEach((table) => {
+            if (initialized.includes(table)) {
+                return;
+            }
+
+            initialized.push(table);
+
+            const tbody = table.querySelector('tbody');
+
+            const makeSortable = (tbody) => {
+                Array.from(tbody.children).forEach((tr, i) => {
+                    tr.querySelectorAll('input, select').forEach((el) => {
+                        el.name = el.name.replace(/\[[0-9]+]/g, '[' + i + ']');
+                    });
+                });
+
+                new Sortables(tbody, {
+                    constrain: true,
+                    opacity: 0.6,
+                    handle: '.drag-handle',
+                    onComplete: function() {
+                        makeSortable(tbody);
+                    }
+                });
+            };
+
+            const addEventsTo = (tr) => {
+                tr.querySelectorAll('button').forEach((bt) => {
+                    const command = bt.dataset.command;
+
+                    switch (command) {
+                        case 'copy':
+                            bt.addEventListener('click', () => {
+                                Backend.getScrollOffset();
+                                const ntr = document.createElement('tr');
+                                Array.from(tr.children).forEach((child) => {
+                                    const next = child.cloneNode(true);
+                                    ntr.appendChild(next);
+                                    const selects = child.querySelectorAll('select');
+                                    const nselects = next.querySelectorAll('select');
+                                    for (let j=0; j<selects.length; j++) {
+                                        nselects[j].value = selects[j].value;
+                                    }
+                                });
+                                tr.parentNode.insertBefore(ntr, tr.nextSibling);
```

Comment:
Why not clone the `tr` element itself?

Reply:
🤷 the original implementation was this way. Changed in 0000907